.appveyor.yml

```diff
@@ -7,12 +7,11 @@ environment:
   APPVEYOR_SAVE_CACHE_ON_ERROR: true
   CLCACHE_SERVER: 1
   PACKAGES: boost-filesystem boost-signals2 boost-test libevent openssl zeromq berkeleydb secp256k1 leveldb
-  PYTHONIOENCODING: utf-8
+  PATH: 'C:\Python37-x64;C:\Python37-x64\Scripts;%PATH%'
+  PYTHONUTF8: 1
```

Comment:
Upgrade to python 3.7 because this require PEP540, make it possible to use utf-8 filename